src/main/java/com/geekbeast/rhizome/pods/hazelcast/BaseHazelcastInstanceConfigurationPod.java

```diff
@@ -46,25 +48,46 @@ public Config getHazelcastConfiguration() {
         return config;
     }
 
-    protected NetworkConfig getNetworkConfig( HazelcastConfiguration hzConfiguration ) {
+    @Bean
```

Comment:
I'm not sure I understand why this shouldn't be exposed this way. Don't I need to do this to be able to @Inject it in the HazelcastPod ?